src/org/isf/operation/gui/OperationRowEdit.java

```diff
@@ -97,7 +96,7 @@ private void fireOperationRowUpdated(OperationRow opeRow) {
 	private JTextArea remarkstextArea ;
 	private JPanel panelButtons;
 	private JLabel lblTransUnite;
-	private JComboBox ResultatcomboBox;
+	private JComboBox<String> ResultatcomboBox;
```

Comment:
Please keep it Java6 compliant for now.
Use English words and standard Java naming (lowercase case initials with Camel case: resultComboBox)